entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh 
+
+if [ "$SUPERUSER_PASSWORD" ]; then
+	/opt/murmur/murmur.x86 -supw $SUPERUSER_PASSWORD
```

Comment:
We'd probably want to change this to still set `-fg -v`. Or even better append to a string of additional arguments and add those to line 7 below.